docs/css/fui-site.css

```diff
@@ -3127,7 +3127,7 @@ This will render 4 boxes spanning 2 cols each indented 2 cols
           height: 100%;
           max-height: inherit; } }
   .fd-app__main {
-    flex: 1 auto;
+    flex: 1 1;
```

Comment:
did you mean to change this? This is automatically generated so if there were any css changes that need to happen you'll need to do them here: https://github.com/SAP/fundamental/tree/master/docs/scss

Reply:
No, I believe this did it automatically with the run build, going to correct now.